src/Microsoft.DotNet.Wpf/src/Themes/PresentationFramework.Fluent/Styles/Frame.xaml

```diff
@@ -7,13 +7,157 @@
 
 <ResourceDictionary xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation" xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml">
 
+    <ControlTemplate x:Key="FrameNavChromeTemplateKey"
+                     TargetType="{x:Type Frame}">
+        <Border Background="{TemplateBinding Background}"
+                BorderBrush="{TemplateBinding BorderBrush}"
+                BorderThickness="{TemplateBinding BorderThickness}"
+                Padding="{TemplateBinding Padding}">
+            <DockPanel>
+                <Grid Background="{StaticResource NavigationWindowNavigationChromeBackground}"
```

Comment:
Check if these static resources like NavigationWindowNavigationChromeBackground , NavigationWindowForwardButtonStyle etc are defined.

Reply:
As discussed these resources are present in `navigationwindow` control which was added as part of the Missing controls [PR](https://github.com/dotnet/wpf/pull/9171).